src/main/java/com/mongodb/jdbc/MongoListCollectionsResult.java

```diff
@@ -1,6 +1,16 @@
 package com.mongodb.jdbc;
 
 public class MongoListCollectionsResult {
+    public static final String TABLE = "table";
+    public static final String COLLECTION = "collection";
+
     public String name;
     public String type;
+
+    public void setType(String type) {
+        // If mongodb type is collection, map it as TABLE.
+        // Otherwise, keep the type as is.
+        this.type = type.equalsIgnoreCase(COLLECTION) ? TABLE : type;
```

Comment:
Given that this class is supposed to represent a collection as described by MongoDB's `listCollections`, it feels a little funny for `type` to take the value `TABLE`, which isn't a concept that exists in MongoDB. I'd suggest the following as an alternative, but happy to discuss further if this isn't idiomatic/desirable for some reason (as I'm far from a java/jdbc expert):

- Leave the `type` value as it is returned from the `listCollections` command
- Add a method (perhaps `getTableType` or similar) that returns the JDBC table type for this collection

Reply:
I agree it's weird and initially we wanted to report 'collection' as is proper. We have to report `table` for `collection` because Tableau thinks that every DB must at least have tables, and it is very likely not the only BI tool to make this assumption 🙃 
I did this here because it is the safest place. It will make sure that there is no place where someone forget to translate.
What I can do is make `type` private and only provide access to `tableType`, where I would do the translation. Or I can keep MongoListCollection as it was before and create MongoListTables where type is the 'jdbc' type. 
While I understand that it might look funny to a Mongo addict, I don't really see the value though of having an extra variable or getter or child class, it is more likely to make things more confusing.

Reply:
As discussed in stand-up, renamed the class to `MongoListTablesResult` to avoid confusion and show that we are exposing the JDBC concept here.